ansible_collections/arista/avd/roles/eos_validate_state/tasks/main.yml

```diff
@@ -73,59 +73,63 @@
 
 - name: Load ntp tasks
   include_tasks: "ntp.yml"
+  when: eos_validate_state_ntp is defined and eos_validate_state_ntp is true
   tags:
     - ntp
 
 - name: Load interface state tasks
   include_tasks: "interface_state.yml"
+  when: eos_validate_state_interface_state is defined and eos_validate_state_interface_state is true
   tags:
     - interfaces_state
 
 - name: Load LLDP Topology tasks when dns domain is defined
   include_tasks: "lldp_topology_fqdn.yml"
   tags:
     - lldp_topology
-  when: dns_domain is defined
+  when: eos_validate_state_lldp_topology is defined and eos_validate_state_lldp_topology is true and dns_domain is defined
```

Comment:
As variables are defined in `default` section, you do not have to run this test `eos_validate_state_* is defined`

Reply:
OK I will fix this